### DISCUSSION TITLE
Configuring runs spawned by asset reconciliation sensor / automaterialization daemon

### DISCUSSION THREAD

**Discussion by dpeng817** [dagster-io/dagster#13958] [Q&A] [answered]
How to provide run config to runs spawned by asset reconciliation sensor?

**Comment by dpeng817** [comment]
https://dagster.slack.com/archives/C01U954MEER/p1682598150898379

**Comment by sryza** [comment]
https://dagster.slack.com/archives/C01U954MEER/p1678783501272329

**Comment by sryza** [comment, marked as answer]
This is not currently possible. We welcome suggestions on what the ideal API would look like for providing this configuration.

**Reply by ei-grad** [reply]
Maybe just add `run_tags` to asset parameters? Treated like the default `run_tags` for runs involving this asset, and be merged with `run_tags` from job/sensor parameters. Some logic may be needed to join assets with equal tags into runs, or may be just declare that each asset materialization should go in its own run.

Is there any discussion about that?

**Reply by sryza** [reply]
Hey @ei-grad - there's an issue where we're tracking potentially adding this: https://github.com/dagster-io/dagster/issues/8979.

**Comment by ei-grad** [comment]
my question is also related:
https://dagster.slack.com/archives/C01U954MEER/p1681372642508839

> Lets assume there are two date-partitioned assets. First asset - the raw data downloaded, which doesn't require a big amount of RAM to be constructed, but should be run with a limited concurrency. Second asset - the processed data, which depends on the raw data asset and can be run without a strict concurrency limitation and requires significant time and a big amount of RAM for processing.
>
> In a single job two assets can be calculated one after other. But with the default multiprocess_executor it is not possible to get different compute configurations in a single job. It could be achived with k8s_job_executor, but we use AWS ECS for which there is no Executor (there is only a EcsRunLauncher). And after all, concurrency limits also don't work on the step basis.
>
> In two separate jobs, we can specify compute configurations and concurrency limits, but there would be no single graph of operations, and some bicycle would be needed to run the second job after the first job completed for corresponding partition.

Currently I ended up with separate `asset_reconciliation_sensor`'s with different `run_tags`.
